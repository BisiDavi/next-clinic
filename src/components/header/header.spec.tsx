import React from 'react'
import { render } from '@testing-library/react'

import Header from './header'

describe('Header Component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Header />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders header page', () => {
    const { getByTestId } = render(<Header />)

    const HEADER = getByTestId('header').textContent

    expect(HEADER).toBe('Header Page')
  })
})
