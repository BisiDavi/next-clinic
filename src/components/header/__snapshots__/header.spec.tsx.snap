// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="header"
      data-testid="header"
    >
      Header Page
    </h1>
  </div>
</DocumentFragment>
`;
